apis/core/v1alpha1/condition.go

```diff
@@ -38,9 +38,9 @@ const (
 	// TypeReferencesResolved resources' references are resolved
 	TypeReferencesResolved ConditionType = "ReferencesResolved"
 
-	// TypeSecretPropagated resources have had connection information propagated
-	// to their secret reference.
-	TypeSecretPropagated ConditionType = "SecretPropagated"
+	// TypeConnectionSecretPropagated resources have had connection information
+	// propagated to their secret reference.
+	TypeConnectionSecretPropagated ConditionType = "SecretPropagated"
```

Comment:
Haha, I think you got the opposite of my intent here, sorry. I was thinking:

```go
TypeSecretPropagated ConditionType = "ConnectionSecretPropagated"
```

i.e. More context for users, less typing for us.

Reply:
Ah! That makes more sense haha